content/java/concepts/strings/terms/subSequence/subSequence.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.subSequence()'
+Description: 'Returns a character sequence from the string'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+# .subSequence() Method
+
+## Descripion
+The `.subSquence()` is a built-in method that returns a new character sequence that is a subsequence of the user-specified string.
+
+The `.subSquence()` method takes two parameters:
+  - `startIndex` - the starting index (inclusive);
+  - `endIndex` - the ending index (exclusive).
+
+  Both parameters must be `integers`.
+
+## Syntax
+```pseudo
+str.subSequence(int startIndex, int endIndex)
+```
+  The `subSequence()` method will return a subsequence of the `str` string that starts with the character at the position `startIndex` and ends with the character at the position `endIndex -1`
+.
+## Example
+
```

Comment:
Include a sentence introducing the example and what it is doing.

Reply:
Hi, I fixed all the suggestions mentioned and I succeeded to add and commit the changes but when I used the command "git push origin rm_branch" i got the following message. Can you please give me a hint?

To https://github.com/rubenm25/docs.git
 ! [rejected]          rm_branch -> rm_branch (non-fast-forward)
error: failed to push some refs to 'https://github.com/rubenm25/docs.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Reply:
You should do a git pull before pushing the changes.

Reply:
I think I managed to push the modified .md file. Can you please check?
Thanks!